src/app/ApplicationMenu.jsx

```diff
@@ -22,19 +22,15 @@ const propTypes = {
 
 const defaultProps = {
   menuHeader: null,
-  navigation: undefined,
+  links: undefined,
```

Comment:
For prop types of array, it always feels better to default it to an empty array as opposed to undefined. Do others have opinions on that?

Reply:
My preference is not to default the array. Defaulting creates an additional state the prop can be in during logic statements which can cause interesting situations.

| Syntax        | if (prop)           |
| ------------- |:-------------:|
| `<Component prop={['1', '2', '3']} />`     | true|
| `<Component />`      | true      |
| `<Component prop={null} />`  | false      |